tests/core/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "asc": "asc",
-    "lint": "eslint . --ext .ts"
+    "lint": "eslint . --ext .ts",
+    "build": "ts-node buildscripts/build.ts"
```

Comment:
I wonder if we could have some kind of ci-build or release-build that runs both linter and compiles the code, and use it in the compile-tests.yml?